src/utils/utils-contour.js

```diff
@@ -166,4 +166,20 @@ export const validateContourState = state => {
       "Minor interval must be a proper divisor of the major interval"
     )
   }
+  if (!data.hasOwnProperty("lat_field") || !data.hasOwnProperty("lon_field")){
+    throw new Error("Latitude and Longitude fields must be provided in data array")
+  }
+}
+
+export const getContourBoundingBox = (data, mapBounds) => {
+  const table = data.table
+  const isGeoPoint = data.is_geo_point_type
+  const latField = isGeoPoint
+    ? `ST_Y(${table}.${data.lat_field})`
+    : `${table}.${data.lat_field}`
```

Comment:
Doesn't _seem_ like it matters here, since you're just doing equality comparisons, but I've been wrapping the lat/lon fields in parens to safeguard against any applied calculations (since `lon - y` will break any calculation using division or multiplication due to order of operations).  I don't really see it being necessary here, but I'm wondering if it's safer just because I'm not sure of the various types of custom expressions users use.

Reply:
👍 done